barretenberg/cpp/.rebuild_patterns

```diff
@@ -1,3 +1,4 @@
 ^barretenberg/.*\\.(cpp|cc|cxx|c\\+\\+|h|hpp|hxx|h\\+\\+|c|h|inl|inc|ipp|tpp|cmake)$
 ^barretenberg/.*CMakeLists\\.txt$
-^barretenberg/.*Dockerfile.*$
\ No newline at end of file
+^barretenberg/.*Dockerfile.*$
```

Comment:
Did you want to add barretenberg/cpp/scripts here?

Reply:
Yes. The stdlib tests were not being re-run despite changing the script that runs them.
Future should be better when tests are listed independently in build manifest.